src/components/Slider/KeenSlider.vue

```diff
@@ -77,7 +77,7 @@ const currentSlide = ref<number>( 0 );
 const [ container, slider ] = useKeenSlider( {
 	...props.sliderOptions,
 	initial: 0,
-	loop: true,
+	loop: false,
```

Comment:
Attention, you now need to set the property to 'true' in all `<BannerSlider>` components in all channes. Maybe change it back to `true` by default, since there is only 1 other component (MultiStepForm) where we need it to be `false`?

Reply:
They don't loop in the old banners